src/csharp/Grpc.Core.Api/AsyncCallState.cs

```diff
@@ -0,0 +1,92 @@
+#region Copyright notice and license
+
+// Copyright 2015 gRPC authors.
```

Comment:
the header should be https://github.com/grpc/grpc/blob/906c12568e9bdf1d8306a2f2ab45efe8beb43f74/src/csharp/Grpc.Core.Api/Properties/AssemblyInfo.cs#L3 for new files